CMakeLists.txt

```diff
@@ -314,3 +314,12 @@ install(
     FILES "${OCIO_PROJECT_CONFIG}" "${OCIO_VERSION_CONFIG}"
     DESTINATION "${OCIO_CONFIG_INSTALL_DIR}"
 )
+
+
+###############################################################################
+# ctest
+
+# To correctly generate the DartConfiguration.tcl, include the CTest CMake file.
+# By doing so 'ctest -F memcheck' works.
+include (CTest)
```

Comment:
[Minor] Just noting that we may move that include up and get rid of the `enable_testing()` call. Could also have this include depend on whether `OCIO_BUILD_TESTS` is `ON` or` OFF` ? I don't think there is issues here though, it just seems a bit of an unusual CMake usage.

Reply:
Not a `cmake` expert so I follow your recommendation.

Reply:
Sorry for having second thoughts on this @hodoulp, but could we add again `enable_testing()` just after the `include (CTest)` ? I think we can now have case where `OCIO_BUILD_TESTS=ON` but the built-in CMake `BUILD_TESTING=OFF` so that the end result is we don't build the tests. I feel like the OCIO variable should always take precedence here.